portal/app/[locale]/_components/navbar/_components/get-started/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Chevron } from 'components/icons/chevron'
+import { Link } from 'components/link'
+import { useNetworkType } from 'hooks/useNetworkType'
+import { useUmami } from 'hooks/useUmami'
+import { usePathname } from 'i18n/navigation'
+import { useTranslations } from 'next-intl'
+import React from 'react'
+
+import { Background } from './background'
+import { HemiLogo } from './hemiLogo'
+
+export const GetStarted = function () {
+  const [networkType] = useNetworkType()
+  const pathname = usePathname()
+  const t = useTranslations('navbar')
+  const { track } = useUmami()
+
+  const href = '/get-started'
+  const onClick = () => track?.('nav - get started', { chain: networkType })
+  const active = pathname.endsWith('/get-started/')
+
+  return (
+    <>
+      <Link
+        className={`shadow-soft font-inter-variable flex cursor-pointer items-center
+        justify-center gap-x-1 rounded-lg border
+        border-neutral-300/55 bg-white py-2 text-sm font-semibold text-neutral-950 md:hidden
+        `}
+        href={href}
+        onClick={track ? onClick : undefined}
+      >
+        <span>{t('get-started')}</span>
+        <Chevron.Right className="[&>path]:fill-neutral-500" />
+      </Link>
+      <Link
+        className={`hidden ${
+          active ? '' : 'cursor-pointer'
+        } group/item md:block`}
+        href={href}
+        onClick={track ? onClick : undefined}
+      >
```

Comment:
Having two links pointing to the same href seems suboptimal and possibly semantically incorrect. Is there a simpler way to do this?

Reply:
one is for mobile and the other for desktop screen